The default font used by VLC does not exists on every system, instead of that,
we use Bitstream Vera, and add it to the dependencies, as anyway many people
have that font installed.

Index: vlc-0.9.0-svn/modules/misc/freetype.c
===================================================================
--- vlc-0.9.0-svn.orig/modules/misc/freetype.c
+++ vlc-0.9.0-svn/modules/misc/freetype.c
@@ -64,8 +64,8 @@
 #define DEFAULT_FONT "" /* Default font found at run-time */
 #define FC_DEFAULT_FONT "Arial"
 #else
-#define DEFAULT_FONT "/usr/share/fonts/truetype/freefont/FreeSerifBold.ttf"
-#define FC_DEFAULT_FONT "Serif Bold"
+#define DEFAULT_FONT "/usr/share/fonts/ttf-bitstream-vera/Vera.ttf"
+#define FC_DEFAULT_FONT "Vera"
 #endif
 
 #if defined(HAVE_FRIBIDI)
